.env.example

```diff
@@ -13,6 +13,11 @@ ADMINISTRATOR_TAG_ID=1014267063788908585
 ADMIN_POINT_REQUEST_CHANNEL=1310614733803618416
 PR_TEMPLATE_ALLOWED_CHANNELS="1270770663048613983,general;1309628262456098878,qa-request;"
 
+# Assign task Forum
+ALLOWED_DEVELOPERS_TAGS='1351647516113768529,Yoel;1351647544916049940,Renz;1351647493359669319,Jose'
```

Comment:
Its weird it have 2 names and 2 codes

also is not a way to select this dynamically? no problem if we have to do it statically, more curious than a real change

Reply:
I agreed, I leave a proposal here, please take a look when you can https://github.com/TeamNovaSoft/novabot/pull/194#discussion_r2006375229